python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2

```diff
@@ -1089,10 +948,156 @@ interface {{ ethernet_interface.name }}
+{%                     set host_mode_cli = host_mode_cli ~ " authenticated" %}
+{%                 endif %}
+   {{ host_mode_cli }}
+{%             endif %}
+{%         endif %}
+{%         if ethernet_interface.dot1x.mac_based_access_list is arista.avd.defined(true) %}
+   dot1x mac based access-list
+{%         endif %}
+{%         if ethernet_interface.dot1x.mac_based_authentication.enabled is arista.avd.defined(true) %}
+{%             if ethernet_interface.dot1x.mac_based_authentication.host_mode_common is arista.avd.defined(true) %}
+   dot1x mac based authentication host-mode common
+{%                 if ethernet_interface.dot1x.mac_based_authentication.always is arista.avd.defined(true) %}
+   dot1x mac based authentication always
+{%                 endif %}
+{%             else %}
+{%                 set auth_cli = "dot1x mac based authentication" %}
+{%                 if ethernet_interface.dot1x.mac_based_authentication.always is arista.avd.defined(true) %}
+{%                     set auth_cli = auth_cli ~ " always" %}
+{%                 endif %}
+   {{ auth_cli }}
+{%             endif %}
+{%         endif %}
+{%         if ethernet_interface.dot1x.timeout is arista.avd.defined %}
+{%             if ethernet_interface.dot1x.timeout.quiet_period is arista.avd.defined %}
+   dot1x timeout quiet-period {{ ethernet_interface.dot1x.timeout.quiet_period }}
+{%             endif %}
+{%             if ethernet_interface.dot1x.timeout.reauth_timeout_ignore is arista.avd.defined(true) %}
+   dot1x timeout reauth-timeout-ignore always
+{%             endif %}
+{%             if ethernet_interface.dot1x.timeout.tx_period is arista.avd.defined %}
+   dot1x timeout tx-period {{ ethernet_interface.dot1x.timeout.tx_period }}
+{%             endif %}
+{%             if ethernet_interface.dot1x.timeout.reauth_period is arista.avd.defined %}
+   dot1x timeout reauth-period {{ ethernet_interface.dot1x.timeout.reauth_period }}
+{%             endif %}
+{%             if ethernet_interface.dot1x.timeout.idle_host is arista.avd.defined %}
+   dot1x timeout idle-host {{ ethernet_interface.dot1x.timeout.idle_host }} seconds
+{%             endif %}
+{%         endif %}
+{%         if ethernet_interface.dot1x.reauthorization_request_limit is arista.avd.defined %}
+   dot1x reauthorization request limit {{ ethernet_interface.dot1x.reauthorization_request_limit }}
+{%         endif %}
+{%         if ethernet_interface.dot1x.unauthorized.access_vlan_membership_egress is arista.avd.defined(true) %}
+   dot1x unauthorized access vlan membership egress
+{%         endif %}
+{%         if ethernet_interface.dot1x.unauthorized.native_vlan_membership_egress is arista.avd.defined(true) %}
+   dot1x unauthorized native vlan membership egress
+{%         endif %}
+{%         if ethernet_interface.dot1x.eapol is arista.avd.defined %}
+{%             if ethernet_interface.dot1x.eapol.disabled is arista.avd.defined(true) %}
+   dot1x eapol disabled
```

Comment:
this is not in correct order
```
transceiver frequency 190000.000
   dot1x pae authenticator
   dot1x authentication failure action traffic allow vlan 101
   dot1x aaa unresponsive action apply cached-results
   dot1x reauthentication
   dot1x port-control auto
   dot1x port-control force-authorized phone
   dot1x host-mode single-host
   dot1x eapol disabled
   dot1x mac based access-list
   dot1x mac based authentication host-mode common
   dot1x mac based authentication always
   dot1x timeout quiet-period 11
   dot1x timeout reauth-timeout-ignore always
   dot1x timeout tx-period 11
   dot1x timeout reauth-period 65
   dot1x timeout idle-host 11 seconds
   dot1x reauthorization request limit 1
   dot1x unauthorized access vlan membership egress
   dot1x unauthorized native vlan membership egress
```

Reply:
I saw `dot1x eapol disabled` and `dot1x eapol authentication failure fallback mba` in nested if condition so thought only one could exist at a time, but we can see now both commands can exist at the same time. 
Fixing the code:

```
   **dot1x eapol disabled**
   dot1x mac based access-list
   dot1x mac based authentication
   dot1x timeout quiet-period 2
   dot1x timeout reauth-timeout-ignore always
   dot1x timeout tx-period 22
   dot1x timeout reauth-period 233
   dot1x timeout idle-host 33 seconds
   dot1x unauthorized access vlan membership egress
   dot1x unauthorized native vlan membership egress
   **dot1x eapol authentication failure fallback mba**
```

Reply:
Fixed!